packages/expo-constants/scripts/source-login-scripts.sh

```diff
@@ -7,9 +7,9 @@
 # script is intended for Xcode build phase scripts on macOS and does not have
 # a shebang so it inherits the current shell.
 
-current_shell=$(ps -cp "$$" -o comm="" | sed s/^-//)
+preferred_shell=$SHELL
```

Comment:
maybe test for $SHELL || $(ps -cp "$$" -o comm="" | sed s/^-//) into preferred shell? then it will resolve the change request too as it stays backwards compatibile